src/main/java/seedu/address/model/person/Name.java

```diff
@@ -16,7 +16,7 @@ public class Name {
      * The first character of the address must not be a whitespace,
      * otherwise " " (a blank string) becomes a valid input.
      */
-    public static final String VALIDATION_REGEX = "[\\p{Alnum}][\\p{Alnum} ]*";
+    public static final String VALIDATION_REGEX = "^[\\p{Alpha}][\\p{Alpha} \\d-/()]*$";
```

Comment:
Same as above about the v1.4 rules. Maybe I should have mentioned this during the meeting today. But perhaps I can ask in the forum about this?